qutebrowser/utils/log.py

```diff
@@ -352,6 +352,9 @@ def qt_message_handler(msg_type, context, msg):
         # may not be available on the system
         "QSslSocket: cannot resolve SSLv3_client_method",
         "QSslSocket: cannot resolve SSLv3_server_method",
+        # When enabling debugging with QtWebEngine
+        "Remote debugging server started successfully. Try pointing a "
+            "Chromium-based browser to ",
```

Comment:
You somehow got a change from `master` into your branch again, in e6d2167085688264e5ee6a81cfd2a7a8f10ded13.

While it's not something terribly bad (as it will be fine after merging), it's kind of confusing, and I have no idea how it happens :laughing: 

Are you trying to update changes from `master` while working on your branch somehow? How do you do that?